### DISCUSSION TITLE
[HttpPut("v1/categories/{id:int}")]

### DISCUSSION THREAD

**Discussion by franciscoguerra** [balta-io/forum#117] [Tira Dúvidas] [answered]
curso -> Fundamentos do ASP.NET 6
Aula -> CRUD de Categorias

Professor estou com esse problema [HttpPut("v1/categories/{id:int}")] , tentei assim também [HttpPut($"v1/categories/{id:int}")] não tive resultado. estou usando o Visual Studio 
![Captura de tela 2022-11-18 201216](https://user-images.githubusercontent.com/12381369/202818281-80594244-af6d-47f2-809c-a0ce370cb06b.png)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @franciscoguerra, qual problema você teve? Tem detalhes do erro?
